chess-AI/mcts.py

```diff
@@ -12,147 +15,125 @@ class Mcts:
         n_values: Dictionary storing the n values
         q_values: Dictionary storing the q values
         exploration: The exploration rate for the tree search
-        p_values: Dictionary storing the search probabilites  
-    """
+        p_values: Dictionary storing the search probabilites"""
```

Comment:
Closing `"""` [should be on newline](https://google.github.io/styleguide/pyguide.html#383-functions-and-methods)